### DISCUSSION TITLE
Disable show URL links between brackets when printing a post

### DISCUSSION THREAD

**Discussion by rfog** [FreshRSS/FreshRSS#6026] [Q&A]
Now I have another problem that I don't know where it comes from. On print, each image or link shows the link as text between brackets []. Is there any way to disable that?

Thanks.

<img width="535" alt="293704113-2c1e8f9b-2a00-4f56-aea4-d3884950acc7" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/4533dced-867b-4d5f-a93b-bec0e2488f4a">

**Comment by Frenzie** [comment]
You mean it doesn't come from the feed?

**Reply by rfog** [reply]
No, at least it is not in the feed. The first image is how it is viewed in the print preview (and printed). Second one the view before printing.

<img width="614" alt="image" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/81ad2f2b-cf74-4247-a5db-ea6296304702">
<img width="555" alt="image" src="https://github.com/FreshRSS/FreshRSS/assets/1958131/174e9857-d883-4e12-bbba-44ba1730c5c3">

What I'm not sure is if it is an Edge/Chrome issue or a FreshRSS one. My custom CSS is:

`h1, h2, span { text-align: center;}
div.content { text-align: justify;}
img { padding: 2px; display: block; margin-left: auto;
margin-right: auto;}
table {
  margin-left: auto;
  margin-right: auto;
}
@media print {
    .flux.not_read {
        display: none;
    }
}`

And disabling the custom css, it shows the same URLs.

**Reply by milinddhamu** [reply]
please show code or data which you're using for this section
